models/attribute-definition/src/attribute-definition-draft/transformers.ts

```diff
@@ -0,0 +1,36 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import { Transformer } from '@commercetools-test-data/core';
+import type {
+  TAttributeDefinitionDraft,
+  TAttributeDefinitionDraftGraphql,
+} from './types';
+
+const transformers = {
+  default: Transformer<TAttributeDefinitionDraft, TAttributeDefinitionDraft>(
+    'default',
+    { buildFields: ['label', 'inputTip', 'type'] }
+  ),
+  rest: Transformer<TAttributeDefinitionDraft, TAttributeDefinitionDraft>(
+    'rest',
+    { buildFields: ['label', 'inputTip', 'type'] }
+  ),
+  graphql: Transformer<
+    TAttributeDefinitionDraft,
+    TAttributeDefinitionDraftGraphql
+  >('graphql', {
+    buildFields: ['type'],
+    addFields: ({ fields }) => {
+      const labelAllLocales = LocalizedString.toLocalizedField(fields.label);
+      const inputTipAllLocales = LocalizedString.toLocalizedField(
+        fields.inputTip
+      );
+      return {
+        __typename: 'AttributeDefinitionDraft',
+        labelAllLocales,
+        inputTipAllLocales,
+      };
+    },
```

Comment:
These lovelies don't exist on the draft 
![image](https://user-images.githubusercontent.com/8136478/217641653-6cb33255-4c9c-46f0-b604-7d01dc53d6e8.png)

Reply:
Awesome - so happy to simplify the graphql transformer!

Reply:
🎸 fixed with (4df0ac4)[https://github.com/commercetools/test-data/pull/192/commits/4df0ac4aa3a69ee761f16e29c58c45606ef214c9]